.github/workflows/lte-integ-test.yml

```diff
@@ -67,12 +67,12 @@ jobs:
           cd lte/gateway
           export MAGMA_DEV_CPUS=3
           export MAGMA_DEV_MEMORY_MB=9216
-          fab integ_test
+          fab integ-test -e --destroy-vm --provision-vm
```

Comment:
why are we doing `-e` (`--echo`) here? if this is intentionally then I suggest to use `--echo`

Reply:
I changed that at the start, but it is unnecessary. Thanks for spotting it, I removed it!